keyvi/bin/keyvi_c/c_api.cpp

```diff
@@ -109,8 +109,12 @@ char* keyvi_dictionary_get_statistics(const keyvi_dictionary* dict) {
   return std_2_c_string(dict->obj_->GetStatistics());
 }
 
-keyvi_match* keyvi_dictionary_get(const keyvi_dictionary* dict, const char* key) {
-  return new keyvi_match(dict->obj_->operator[](key));
+keyvi_match* keyvi_dictionary_get(const keyvi_dictionary* dict, const keyvi_bytes key) {
```

Comment:
now that I see the API, I realize we don't need to wrap and unwrap it:

```
keyvi_match* keyvi_dictionary_get(const keyvi_dictionary* dict, const char* key, size_t length)
```

Reply:
good point! done in 6073fdd